swap_meet/vendor.py

```diff
@@ -1,2 +1,85 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        if inventory is None:
+            inventory = []
+        self.inventory = inventory
+    
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+    
+    def remove(self, item):
+        if not item in self.inventory:
+            return False
+        else:
```

Comment:
Since you `return False` above, you can get away with removing this `else:`. Then the `if` statement just becomes a guard clause.